src/library/scala/collection/mutable/ListBuffer.scala

```diff
@@ -61,6 +62,7 @@ class ListBuffer[A]
   // Avoids copying where possible.
   override def toList: List[A] = {
     aliased = nonEmpty
+    ScalaRunTime.releaseFence()
```

Comment:
Why no fence for the other methods that mutate? I see they're private, but they are called from public methods, I assume?